mteb/tasks/BitextMining/multilingual/RomaTalesBitextMining.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskBitextMining
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class RomaTalesBitextMining(AbsTaskBitextMining):
+    metadata = TaskMetadata(
+        name="RomaTalesBitextMining",
+        dataset={
+            "path": "kardosdrur/roma-tales",
+            "revision": "f4394dbca6845743cd33eba77431767b232ef489",
+        },
+        description="Parallel corpus of Roma Tales in Lovari with Hungarian translations.",
+        reference="https://idoc.pub/documents/idocpub-zpnxm9g35ylv",
+        type="BitextMining",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["rom-Latn", "hun-Latn"],
+        main_score="f1",
+        date=None,  # Unknown, these are folk tales
+        form=["written"],
+        domains=["Fiction"],
+        task_subtypes=None,  # Didn't fit any
```

Comment:
```suggestion
        task_subtypes=[],
```
If you feel like one is missing please add it. Otherwise an empty list is good.